.github/workflows/publish-v2.yml

```diff
@@ -86,7 +86,7 @@ jobs:
         run: npx vsce package
 
       - name: federated login
-        uses: azure/login@v2
+        uses: azure/login@a65d910e8af852a8061c627c456678983e180302 # v2.2.0
```

Comment:
💡 Can we please add the cheangelog and the release part back into this :) from the current `publics.yaml` we can rename this file latter once we retire current release process.

thank you 

![](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExOHczeTV4NTl3MDB3Mjg5eTd6Zzc3bGduZmt2ZXpmbG12MWx1YTlqYyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/ZvdE6zT9AnCnZCDguo/giphy.gif)